Update with latest changes for version 1.1.0 of the framework (#12845)
DataDog/documentation: content/en/continuous_integration/setup_tests/swift.md

* macOS >= 10.13
* tvOS >= 11.0

> **Note**: If you are using Swift Concurrency, Xcode >= 13.2 is needed for precise span representation of asynchronous tasks </div>

Reviewer: ```suggestion
**Note**: If you are using Swift Concurrency, you need Xcode >= 13.2 for precise span representation of asynchronous tasks.
```

**Example**: `true`

`DATADOG_CLIENT_TOKEN`

> : The [Datadog Client Token][1] to use to report test results. Alternatively you can also use an API key here<br/>

Reviewer: ```suggestion
: Use the [Datadog Client Token][1] to report test results. Alternatively, use an API key.<br/>
```

`DD_MAX_PAYLOAD_SIZE`
: Sets the maximum size reported from the payload. Default `1024` (Integer)

`DD_DISABLE_NETWORK_CALL_STACK`

> : Disables the callstack information in the network spans (Boolean)

Reviewer: ```suggestion
: Disables the call stack information in the network spans (Boolean)
```

`DD_ENABLE_NETWORK_CALL_STACK_SYMBOLICATED`

> : Shows the callstack information with precise file and line information, not only the method name. It can impact the performance of the tests. (Boolean)

Reviewer: ```suggestion
: Shows the call stack information with not only the method name, but also the precise file and line information. May impact your tests' performance (Boolean)
```
